subsystem/ams_subscriptions_test.go

```diff
@@ -326,16 +326,6 @@ var _ = Describe("test AMS subscriptions", func() {
 		// ATTENTION: this test override a wiremock stub - DO NOT RUN IN PARALLEL
 		It("UpdateSubscription failed", func() {
 
-			waitForInstallationPreparationCompletionStatus := func(clusterID strfmt.UUID, status string) {
```

Comment:
How is this relevant to the PR?

Reply:
I needed to extract it from inside a specific test so another test can use it.
It was a private variable of a specific test and not it is a common function accessible for other subsystem test - specifically for the new test I wrote.